tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/admin/ProjectPayload.scala

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.bluebrain.nexus.tests.admin
+
+import ch.epfl.bluebrain.nexus.testkit.Generators
+import ch.epfl.bluebrain.nexus.tests.config.TestsConfig
+import io.circe.syntax.KeyOps
+import io.circe.{Encoder, Json, JsonObject}
+
+final case class ProjectPayload(
```

Comment:
Introducing this as it makes it easier to manage projects in the tests, as `enforceSchema` introduces some extra complexity

Reply:
I'm not against this as such, but payloads are fundamentally JSON. I wonder if you could just have methods which returned JSON instead

Reply:
I tried the JSON approach first but as we create projects in almost every integration test and that projects have a strict shape (and relatively simple), I found that this one gave better results